.github/workflows/dependabot-auto-evaulate-pr.yaml

```diff
@@ -0,0 +1,31 @@
+name: Dependabot Evaluation
+
+on:
+    pull_request:
+        paths:
+            - "cookbook/**/*"
```

Comment:
I don't think we want this to run on all pull requests, perhaps a `cron` would be more appropriate?

Reply:
Checking the username / branch upon a new pull request is a great idea.